docs/ddl-of-carbondata.md

```diff
@@ -83,7 +83,6 @@ CarbonData DDL statements are documented here,which includes:
 
 | Property                                                     | Description                                                  |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| [DICTIONARY_INCLUDE](#dictionary-encoding-configuration)     | Columns for which dictionary needs to be generated           |
 | [NO_INVERTED_INDEX](#inverted-index-configuration)           | Columns to exclude from inverted index generation            |
 | [INVERTED_INDEX](#inverted-index-configuration)              | Columns to include for inverted index generation             |
```

Comment:
no need

Reply:
inverted index is not removed, it is required